### ISSUE TITLE
AMP: running Automatic Mixed Precision (AMP) checks with YOLOv8n...

### ISSUE THREAD

**Issue by meixiangzhang** [ultralytics/ultralytics#7318]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

When I was training YOLOv8s, he still prompted "AMP: running Automatic Mixed Precision (AMP) checks with YOLOv8n...
AMP: checks passed ✅
train: Scanning F:\More\ultralytics-main\ultralytics\datasets\VOC2007\labels... 4058 images, 0 backgrounds, 0 corrupt: 100%|██████████| 4058/4058 [00:01<00:00, 3447.75it/s]
train: New cache created: F:\More\ultralytics-main\ultralytics\datasets\VOC2007\labels.cache
val: Scanning F:\More\ultralytics-main\ultralytics\datasets\VOC2007\labels... 451 images, 0 backgrounds, 0 corrupt: 100%|██████████| 451/451 [00:00<00:00, 3089.07it/s]
val: New cache created: F:\More\ultralytics-main\ultralytics\datasets\VOC2007\labels.cache
". Will this affect accuracy? Will only the training set data be used during training? Will the validation set data be used during evaluation? Should the test set be set?

**Comment by meixiangzhang** [comment]
`Administrator@YFX-20220331ZER MINGW64 /f/More/ultralytics-main
$ **yolo detect train model=yolov5su.pt**
WARNING ⚠️ 'data' is missing. Using default 'data=../voc.yaml'.
Ultralytics YOLOv8.0.234 � Python-3.8.18 torch-1.10.1 CUDA:0 (NVIDIA GeForce RTX 3090, 24576MiB)
WARNING ⚠️ Upgrade to torch>=2.0.0 for deterministic training.
engine\trainer: task=detect, mode=train, model=yolov5su.pt, data=../voc.yaml, epochs=100, time=None, patience=50, batch=16, imgsz=640, save=True, save_period=-1, cache=False, device=No
ne, workers=4, project=None, name=train9, exist_ok=False, pretrained=True, optimizer=auto, verbose=True, seed=0, deterministic=True, single_cls=False, rect=False, cos_lr=False, close_m
osaic=10, resume=False, amp=True, fraction=1.0, profile=False, freeze=None, overlap_mask=True, mask_ratio=4, dropout=0.0, val=True, split=val, save_json=False, save_hybrid=False, conf=
None, iou=0.7, max_det=300, half=False, dnn=False, plots=True, source=None, vid_stride=1, stream_buffer=False, visualize=False, augment=False, agnostic_nms=False, classes=None, retina_
masks=False, embed=None, show=False, save_frames=False, save_txt=False, save_conf=False, save_crop=False, show_labels=True, show_conf=True, show_boxes=True, line_width=None, format=tor
chscript, keras=False, optimize=False, int8=False, dynamic=False, simplify=False, opset=None, workspace=4, nms=False, lr0=0.01, lrf=0.01, momentum=0.937, weight_decay=0.0005, warmup_ep
ochs=3.0, warmup_momentum=0.8, warmup_bias_lr=0.1, box=7.5, cls=0.5, dfl=1.5, pose=12.0, kobj=1.0, label_smoothing=0.0, nbs=64, hsv_h=0.015, hsv_s=0.7, hsv_v=0.4, degrees=0.0, translat
e=0.1, scale=0.5, shear=0.0, perspective=0.0, flipud=0.0, fliplr=0.5, mosaic=1.0, mixup=0.0, copy_paste=0.0, auto_augment=randaugment, erasing=0.4, crop_fraction=1.0, cfg=None, tracker
=botsort.yaml, save_dir=runs\detect\train9
Overriding model.yaml nc=80 with nc=20

                   from  n    params  module                                       arguments
  0                  -1  1      3520  ultralytics.nn.modules.conv.Conv             [3, 32, 6, 2, 2]
  1                  -1  1     18560  ultralytics.nn.modules.conv.Conv             [32, 64, 3, 2]
  2                  -1  1     18816  ultralytics.nn.modules.block.C3              [64, 64, 1]
  3                  -1  1     73984  ultralytics.nn.modules.conv.Conv             [64, 128, 3, 2]
  4                  -1  2    115712  ultralytics.nn.modules.block.C3              [128, 128, 2]
  5                  -1  1    295424  ultralytics.nn.modules.conv.Conv             [128, 256, 3, 2]
  6                  -1  3    625152  ultralytics.nn.modules.block.C3              [256, 256, 3]
  7                  -1  1   1180672  ultralytics.nn.modules.conv.Conv             [256, 512, 3, 2]
  8                  -1  1   1182720  ultralytics.nn.modules.block.C3              [512, 512, 1]
  9                  -1  1    656896  ultralytics.nn.modules.block.SPPF            [512, 512, 5]
 10                  -1  1    131584  ultralytics.nn.modules.conv.Conv             [512, 256, 1, 1]
 11                  -1  1         0  torch.nn.modules.upsampling.Upsample         [None, 2, 'nearest']
 12             [-1, 6]  1         0  ultralytics.nn.modules.conv.Concat           [1]
 13                  -1  1    361984  ultralytics.nn.modules.block.C3              [512, 256, 1, False]
 14                  -1  1     33024  ultralytics.nn.modules.conv.Conv             [256, 128, 1, 1]
 15                  -1  1         0  torch.nn.modules.upsampling.Upsample         [None, 2, 'nearest']
 16             [-1, 4]  1         0  ultralytics.nn.modules.conv.Concat           [1]
 17                  -1  1     90880  ultralytics.nn.modules.block.C3              [256, 128, 1, False]
 18                  -1  1    147712  ultralytics.nn.modules.conv.Conv             [128, 128, 3, 2]
 19            [-1, 14]  1         0  ultralytics.nn.modules.conv.Concat           [1]
 20                  -1  1    296448  ultralytics.nn.modules.block.C3              [256, 256, 1, False]
 21                  -1  1    590336  ultralytics.nn.modules.conv.Conv             [256, 256, 3, 2]
 22            [-1, 10]  1         0  ultralytics.nn.modules.conv.Concat           [1]
 23                  -1  1   1182720  ultralytics.nn.modules.block.C3              [512, 512, 1, False]
 24        [17, 20, 23]  1   2123788  ultralytics.nn.modules.head.Detect           [20, [128, 256, 512]]
YOLOv5s summary: 262 layers, 9129932 parameters, 9129916 gradients, 24.1 GFLOPs

Transferred 421/427 items from pretrained weights
Freezing layer 'model.24.dfl.conv.weight'
**AMP: running Automatic Mixed Precision (AMP) checks with YOLOv8n...**
AMP: checks passed ✅
train: Scanning F:\More\ultralytics-main\ultralytics\datasets\VOC2007\labels... 4058 images, 0 backgrounds, 0 corrupt: 100%|██████████| 4058/4058 [00:01<00:00, 3471.34it/s]
train: New cache created: F:\More\ultralytics-main\ultralytics\datasets\VOC2007\labels.cache
val: Scanning F:\More\ultralytics-main\ultralytics\datasets\VOC2007\labels... 451 images, 0 backgrounds, 0 corrupt: 100%|██████████| 451/451 [00:00<00:00, 2818.75it/s]
val: New cache created: F:\More\ultralytics-main\ultralytics\datasets\VOC2007\labels.cache
Plotting labels to runs\detect\train9\labels.jpg...
optimizer: 'optimizer=auto' found, ignoring 'lr0=0.01' and 'momentum=0.937' and determining best 'optimizer', 'lr0' and 'momentum' automatically...
optimizer: AdamW(lr=0.000417, momentum=0.9) with parameter groups 69 weight(decay=0.0), 76 weight(decay=0.0005), 75 bias(decay=0.0)
100 epochs...

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size
      1/100      3.98G     0.9903      1.842      1.244         37        640: 100%|██████████| 254/254 [00:33<00:00,  7.69it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 15/15 [00:01<00:00, 10.10it/s]
                   all        451       1120      0.656      0.735      0.764      0.549
`
In the bold part of the above text, it can be seen that the model I trained is yolov5su.pt, and below it is a prompt of "AMP: running Automatic Mixed Precision (AMP) checks with YOLOv8n...". Will this affect mAP? If I want to modify it, what should I do?

**Comment by glenn-jocher** [comment]
@meixiangzhang it appears there's a bit of confusion in the command you've used. The model name `yolov5su.pt` suggests a YOLOv5 model, but you're running it in a YOLOv8 environment. The AMP check message mentioning YOLOv8n is a standard output indicating that AMP is enabled and working correctly, which is beneficial for faster training with mixed precision without significantly affecting the model's accuracy or mAP.

If you're training with YOLOv8 and want to ensure everything is set up correctly, make sure to use a YOLOv8 model and the corresponding configuration files. The AMP checks are a part of the training process and should not negatively impact your mAP. If you want to train without AMP, you can disable it, but typically, it's recommended to use AMP for the performance benefits it provides. For further guidance on training with YOLOv8, please refer to our documentation.

**Comment by meixiangzhang** [comment]
@glenn-jocher Thank you for your answer! I will recheck my environment and then try again.

**Comment by glenn-jocher** [comment]
@meixiangzhang You're welcome! If you have any more questions or need further assistance after rechecking your environment, feel free to reach out. Happy training! 🚀
